### ISSUE TITLE
Journey admin menu: clarity on what's public and what not

### ISSUE THREAD

**Issue by SimoneZaza** [alkem-io/client-web#4906]
## Description

As an admin, I need to be aware of the things that are publicly visible, even though it is a private Space. 

### General
- [ ] Change order of settings: Move "Settings" to be last in the list (after Storage)

### Profile tab
- [ ] Change description on top: 
- Here you can edit the public profile for this Space(/Challenge/Opportunity). This information is also visible to non-members.
- [ ] Add description to the References block, underneath or next to the title:
- Please keep in mind that all the files you attach here become publicly available. If you only want the members of this Space to be able to access the files, attach it to a post or a collection of links on one of the member pages.

### Context tab
- [ ] Change description on top: 
- Here you can edit the context for this Space(/Challenge/Opportunity), which is publicly visible. This information is also visible to non-members.

### Settings tab
- [ ] Change the description of the Private option in the visibility section:
- Private: this Space is visible to everyone on the platform, but the content is only visible to members. Please note that both the information in the profile and the context are publicly visible.

**Comment by SimoneZaza** [comment]
@techsmyth, I think we actually need to have different labels for Challenges and Opportunities? 🫤 Something in which it states that depending on the visibility of the Space/Challenge, this information is or isn't publicly visible?

**Comment by techsmyth** [comment]
@SimoneZaza actually the same situation arises for challenges: the profile is visible even if it is private. But in that case it is only visible for space members if the space is private or if the space is public for all users

**Comment by SimoneZaza** [comment]
@techsmyth yes, that is indeed what I meant. Do we want to explain all this? I guess so.. will be a bit messy though..

**Comment by techsmyth** [comment]
Or we say something along the lines of "This profile is visible for users that can see the Challenge exists"
